source/databricks/calculation_engine/tests/entry_points/test_entry_points.py

```diff
@@ -40,6 +40,7 @@ def assert_entry_point_exists(entry_point_name: str) -> Any:
     [
         "start_calculator",
         "migrate_data_lake",
+        "optimise_tables",
     ],
 )
 def test__entry_point__start_calculator__can_load_entry_point(
```

Comment:
The test name is wrong.
```suggestion
def test__installed_package__can_load_entry_point(
```